Readme.md

```diff
@@ -96,6 +80,33 @@ As this tool does not provide any schema validation, it is important to note tha
 Any typo in the parameter `category` or the header column of the csv-file is directly translated into the
 dedicated XML-tag, leading to potential errors when attempting to process the XML-file further.
 
+Installation
+------------
+
+You can [download a binary executable](https://github.com/blue-yonder/di-csv2xml/releases) (for
+64Bit OS-X, Windows and Linux) or build it yourself using:
+
+```bash
+git clone https://github.com/blue-yonder/di-csv2xml.git
+cd di-csv2xml
+cargo build --release
+```
+
+You will find the executable in the `.\target\release\` directory.
+
+You can install cargo from [here](https://rustup.rs) if it is not installed.
+
+Benchmarks
+----------
+
+This tool uses [criterion](https://github.com/bheisler/criterion.rs) for benchmarking. You can execute the benchmarks with:
+
+```shell
+cargo bench
+```
```

Comment:
Execute in which directory? I assume the sources have to be compiled first.

Reply:
in the repositories direcotry. `cargo` will compile them automatically if they have changed. In release mode.